Expecto fails to build after 'paket update'

@adamchester @cloudRoutine 

Doing`paket update` with paket versions between 4.1 and 5.0-alpha.6 all fail with:

```
/usr/local/share/dotnet/sdk/1.0.1/Microsoft.Common.CurrentVersion.targets(2628,7): error MSB4057: The target "CreateManifestResourceNames" does not exist in the project. [/Users/h/dev/haf/expecto/Expecto.netcore/Expecto.netcore.fsproj]
```

Here's the diff generated:

```diff
diff --git a/Expecto.BenchmarkDotNet/Expecto.BenchmarkDotNet.fsproj b/Expecto.BenchmarkDotNet/Expecto.BenchmarkDotNet.fsproj
index 733a1da..8706a3d 100644
--- a/Expecto.BenchmarkDotNet/Expecto.BenchmarkDotNet.fsproj
+++ b/Expecto.BenchmarkDotNet/Expecto.BenchmarkDotNet.fsproj
@@ -1,4 +1,4 @@
-<?xml version="1.0" encoding="utf-8" standalone="yes"?>
+﻿<?xml version="1.0" encoding="utf-8" standalone="yes"?>
 <Project ToolsVersion="14.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
   <PropertyGroup>
     <Name>Expecto.BenchmarkDotNet</Name>
@@ -36,6 +36,12 @@
     <Reference Include="System.Numerics" />
   </ItemGroup>
   <ItemGroup>
+    <Content Include="any\netstandard1.6\FSharp.Core.optdata">
+      <Paket>True</Paket>
+    </Content>
+    <Content Include="any\netstandard1.6\FSharp.Core.sigdata">
+      <Paket>True</Paket>
+    </Content>
     <ProjectReference Include="../Expecto/Expecto.fsproj">
       <Name>Expecto.fsproj</Name>
       <Project>{c0d55728-10a9-4a7a-9df9-d2f21f663ac2}</Project>
@@ -65,7 +71,7 @@
     <When Condition="$(TargetFrameworkIdentifier) == '.NETFramework' And $(TargetFrameworkVersion) == 'v4.6.1'">
       <ItemGroup>
         <Reference Include="BenchmarkDotNet">
-          <HintPath>..\packages\BenchmarkDotNet\lib\net45\BenchmarkDotNet.dll</HintPath>
+          <HintPath>..\packages\BenchmarkDotNet\lib\net46\BenchmarkDotNet.dll</HintPath>
           <Private>True</Private>
           <Paket>True</Paket>
         </Reference>
@@ -76,7 +82,7 @@
     <When Condition="$(TargetFrameworkIdentifier) == '.NETFramework' And $(TargetFrameworkVersion) == 'v4.6.1'">
       <ItemGroup>
         <Reference Include="BenchmarkDotNet.Core">
-          <HintPath>..\packages\BenchmarkDotNet.Core\lib\net45\BenchmarkDotNet.Core.dll</HintPath>
+          <HintPath>..\packages\BenchmarkDotNet.Core\lib\net46\BenchmarkDotNet.Core.dll</HintPath>
           <Private>True</Private>
           <Paket>True</Paket>
         </Reference>
@@ -87,7 +93,7 @@
     <When Condition="$(TargetFrameworkIdentifier) == '.NETFramework' And $(TargetFrameworkVersion) == 'v4.6.1'">
       <ItemGroup>
         <Reference Include="BenchmarkDotNet.Toolchains.Roslyn">
-          <HintPath>..\packages\BenchmarkDotNet.Toolchains.Roslyn\lib\net45\BenchmarkDotNet.Toolchains.Roslyn.dll</HintPath>
+          <HintPath>..\packages\BenchmarkDotNet.Toolchains.Roslyn\lib\net46\BenchmarkDotNet.Toolchains.Roslyn.dll</HintPath>
           <Private>True</Private>
           <Paket>True</Paket>
         </Reference>
@@ -98,7 +104,7 @@
     <When Condition="$(TargetFrameworkIdentifier) == '.NETFramework' And $(TargetFrameworkVersion) == 'v4.6.1'">
       <ItemGroup>
         <Reference Include="FSharp.Core">
-          <HintPath>..\packages\FSharp.Core\lib\net40\FSharp.Core.dll</HintPath>
+          <HintPath>..\packages\FSharp.Core\lib\net45\FSharp.Core.dll</HintPath>
           <Private>True</Private>
           <Paket>True</Paket>
         </Reference>
@@ -130,6 +136,28 @@
   <Choose>
     <When Condition="$(TargetFrameworkIdentifier) == '.NETFramework' And $(TargetFrameworkVersion) == 'v4.6.1'">
       <ItemGroup>
+        <Reference Include="Microsoft.DotNet.InternalAbstractions">
+          <HintPath>..\packages\Microsoft.DotNet.InternalAbstractions\lib\net451\Microsoft.DotNet.InternalAbstractions.dll</HintPath>
+          <Private>True</Private>
+          <Paket>True</Paket>
+        </Reference>
+      </ItemGroup>
+    </When>
+  </Choose>
+  <Choose>
+    <When Condition="$(TargetFrameworkIdentifier) == '.NETFramework' And $(TargetFrameworkVersion) == 'v4.6.1'">
+      <ItemGroup>
+        <Reference Include="Microsoft.DotNet.PlatformAbstractions">
+          <HintPath>..\packages\Microsoft.DotNet.PlatformAbstractions\lib\net451\Microsoft.DotNet.PlatformAbstractions.dll</HintPath>
+          <Private>True</Private>
+          <Paket>True</Paket>
+        </Reference>
+      </ItemGroup>
+    </When>
+  </Choose>
+  <Choose>
+    <When Condition="$(TargetFrameworkIdentifier) == '.NETFramework' And $(TargetFrameworkVersion) == 'v4.6.1'">
+      <ItemGroup>
         <Reference Include="System.AppContext">
           <HintPath>..\packages\System.AppContext\lib\net46\System.AppContext.dll</HintPath>
           <Private>True</Private>
@@ -379,4 +407,4 @@
       </ItemGroup>
     </When>
   </Choose>
-</Project>
+</Project>
\ No newline at end of file
diff --git a/Expecto.FsCheck/Expecto.FsCheck.fsproj b/Expecto.FsCheck/Expecto.FsCheck.fsproj
index c3ddb4e..16340e8 100644
--- a/Expecto.FsCheck/Expecto.FsCheck.fsproj
+++ b/Expecto.FsCheck/Expecto.FsCheck.fsproj
@@ -1,4 +1,4 @@
-<?xml version="1.0" encoding="utf-8"?>
+﻿<?xml version="1.0" encoding="utf-8"?>
 <Project ToolsVersion="4.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
   <PropertyGroup>
     <Configuration Condition=" '$(Configuration)' == '' ">Debug</Configuration>
@@ -48,6 +48,12 @@
   </Choose>
   <Import Project="$(FSharpTargetsPath)" Condition="Exists('$(FSharpTargetsPath)')" />
   <ItemGroup>
+    <Content Include="any\netstandard1.6\FSharp.Core.optdata">
+      <Paket>True</Paket>
+    </Content>
+    <Content Include="any\netstandard1.6\FSharp.Core.sigdata">
+      <Paket>True</Paket>
+    </Content>
     <Compile Include="FsCheck.fs" />
     <Content Include="paket.references" />
   </ItemGroup>
@@ -78,11 +84,22 @@
     <When Condition="$(TargetFrameworkIdentifier) == '.NETFramework' And $(TargetFrameworkVersion) == 'v4.6.1'">
       <ItemGroup>
         <Reference Include="FSharp.Core">
-          <HintPath>..\packages\FSharp.Core\lib\net40\FSharp.Core.dll</HintPath>
+          <HintPath>..\packages\FSharp.Core\lib\net45\FSharp.Core.dll</HintPath>
           <Private>True</Private>
           <Paket>True</Paket>
         </Reference>
       </ItemGroup>
     </When>
   </Choose>
-</Project>
+  <Choose>
+    <When Condition="$(TargetFrameworkIdentifier) == '.NETFramework' And $(TargetFrameworkVersion) == 'v4.6.1'">
+      <ItemGroup>
+        <Reference Include="System.ValueTuple">
+          <HintPath>..\packages\System.ValueTuple\lib\netstandard1.0\System.ValueTuple.dll</HintPath>
+          <Private>True</Private>
+          <Paket>True</Paket>
+        </Reference>
+      </ItemGroup>
+    </When>
+  </Choose>
+</Project>
\ No newline at end of file
diff --git a/Expecto.Sample/App.config b/Expecto.Sample/App.config
index 3918676..91448d2 100644
--- a/Expecto.Sample/App.config
+++ b/Expecto.Sample/App.config
@@ -7,17 +7,17 @@
   <dependentAssembly>
     <Paket>True</Paket>
     <assemblyIdentity name="FSharp.Core" publicKeyToken="b03f5f7f11d50a3a" culture="neutral" />
-    <bindingRedirect oldVersion="0.0.0.0-65535.65535.65535.65535" newVersion="4.3.1.0" />
+    <bindingRedirect oldVersion="0.0.0.0-65535.65535.65535.65535" newVersion="4.4.1.0" />
   </dependentAssembly>
   <dependentAssembly>
     <Paket>True</Paket>
     <assemblyIdentity name="Microsoft.CodeAnalysis" publicKeyToken="31bf3856ad364e35" culture="neutral" />
-    <bindingRedirect oldVersion="0.0.0.0-65535.65535.65535.65535" newVersion="2.0.0.0" />
+    <bindingRedirect oldVersion="0.0.0.0-65535.65535.65535.65535" newVersion="2.1.0.0" />
   </dependentAssembly>
   <dependentAssembly>
     <Paket>True</Paket>
     <assemblyIdentity name="Microsoft.CodeAnalysis.CSharp" publicKeyToken="31bf3856ad364e35" culture="neutral" />
-    <bindingRedirect oldVersion="0.0.0.0-65535.65535.65535.65535" newVersion="2.0.0.0" />
+    <bindingRedirect oldVersion="0.0.0.0-65535.65535.65535.65535" newVersion="2.1.0.0" />
   </dependentAssembly>
   <dependentAssembly>
     <Paket>True</Paket>
diff --git a/Expecto.Sample/Expecto.Sample.fsproj b/Expecto.Sample/Expecto.Sample.fsproj
index ac2d948..721e384 100644
--- a/Expecto.Sample/Expecto.Sample.fsproj
+++ b/Expecto.Sample/Expecto.Sample.fsproj
@@ -1,4 +1,4 @@
-<?xml version="1.0" encoding="utf-8" standalone="yes"?>
+﻿<?xml version="1.0" encoding="utf-8" standalone="yes"?>
 <Project ToolsVersion="14.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
   <PropertyGroup>
     <Name>Expecto.Sample</Name>
@@ -37,6 +37,12 @@
     <Reference Include="System.Numerics" />
   </ItemGroup>
   <ItemGroup>
+    <Content Include="any\netstandard1.6\FSharp.Core.optdata">
+      <Paket>True</Paket>
+    </Content>
+    <Content Include="any\netstandard1.6\FSharp.Core.sigdata">
+      <Paket>True</Paket>
+    </Content>
     <ProjectReference Include="../Expecto/Expecto.fsproj">
       <Name>Expecto.fsproj</Name>
       <Project>{c0d55728-10a9-4a7a-9df9-d2f21f663ac2}</Project>
@@ -75,7 +81,7 @@
     <When Condition="$(TargetFrameworkIdentifier) == '.NETFramework' And $(TargetFrameworkVersion) == 'v4.6.1'">
       <ItemGroup>
         <Reference Include="BenchmarkDotNet.Core">
-          <HintPath>..\packages\BenchmarkDotNet.Core\lib\net45\BenchmarkDotNet.Core.dll</HintPath>
+          <HintPath>..\packages\BenchmarkDotNet.Core\lib\net46\BenchmarkDotNet.Core.dll</HintPath>
           <Private>True</Private>
           <Paket>True</Paket>
         </Reference>
@@ -86,7 +92,29 @@
     <When Condition="$(TargetFrameworkIdentifier) == '.NETFramework' And $(TargetFrameworkVersion) == 'v4.6.1'">
       <ItemGroup>
         <Reference Include="FSharp.Core">
-          <HintPath>..\packages\FSharp.Core\lib\net40\FSharp.Core.dll</HintPath>
+          <HintPath>..\packages\FSharp.Core\lib\net45\FSharp.Core.dll</HintPath>
+          <Private>True</Private>
+          <Paket>True</Paket>
+        </Reference>
+      </ItemGroup>
+    </When>
+  </Choose>
+  <Choose>
+    <When Condition="$(TargetFrameworkIdentifier) == '.NETFramework' And $(TargetFrameworkVersion) == 'v4.6.1'">
+      <ItemGroup>
+        <Reference Include="Microsoft.DotNet.InternalAbstractions">
+          <HintPath>..\packages\Microsoft.DotNet.InternalAbstractions\lib\net451\Microsoft.DotNet.InternalAbstractions.dll</HintPath>
+          <Private>True</Private>
+          <Paket>True</Paket>
+        </Reference>
+      </ItemGroup>
+    </When>
+  </Choose>
+  <Choose>
+    <When Condition="$(TargetFrameworkIdentifier) == '.NETFramework' And $(TargetFrameworkVersion) == 'v4.6.1'">
+      <ItemGroup>
+        <Reference Include="Microsoft.DotNet.PlatformAbstractions">
+          <HintPath>..\packages\Microsoft.DotNet.PlatformAbstractions\lib\net451\Microsoft.DotNet.PlatformAbstractions.dll</HintPath>
           <Private>True</Private>
           <Paket>True</Paket>
         </Reference>
@@ -104,4 +132,15 @@
       </ItemGroup>
     </When>
   </Choose>
-</Project>
+  <Choose>
+    <When Condition="$(TargetFrameworkIdentifier) == '.NETFramework' And $(TargetFrameworkVersion) == 'v4.6.1'">
+      <ItemGroup>
+        <Reference Include="System.ValueTuple">
+          <HintPath>..\packages\System.ValueTuple\lib\netstandard1.0\System.ValueTuple.dll</HintPath>
+          <Private>True</Private>
+          <Paket>True</Paket>
+        </Reference>
+      </ItemGroup>
+    </When>
+  </Choose>
+</Project>
\ No newline at end of file
diff --git a/Expecto.Tests.CSharp/Expecto.Tests.CSharp.csproj b/Expecto.Tests.CSharp/Expecto.Tests.CSharp.csproj
index 1aa2ef7..548e7c6 100644
--- a/Expecto.Tests.CSharp/Expecto.Tests.CSharp.csproj
+++ b/Expecto.Tests.CSharp/Expecto.Tests.CSharp.csproj
@@ -1,4 +1,4 @@
-<?xml version="1.0" encoding="utf-8"?>
+﻿<?xml version="1.0" encoding="utf-8"?>
 <Project ToolsVersion="14.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
   <Import Project="$(MSBuildExtensionsPath)\$(MSBuildToolsVersion)\Microsoft.Common.props" Condition="Exists('$(MSBuildExtensionsPath)\$(MSBuildToolsVersion)\Microsoft.Common.props')" />
   <PropertyGroup>
@@ -46,6 +46,12 @@
     <Reference Include="System.Xml" />
   </ItemGroup>
   <ItemGroup>
+    <Content Include="any\netstandard1.6\FSharp.Core.optdata">
+      <Paket>True</Paket>
+    </Content>
+    <Content Include="any\netstandard1.6\FSharp.Core.sigdata">
+      <Paket>True</Paket>
+    </Content>
     <Compile Include="Tests.cs" />
   </ItemGroup>
   <ItemGroup>
@@ -70,11 +76,22 @@
     <When Condition="$(TargetFrameworkIdentifier) == '.NETFramework' And $(TargetFrameworkVersion) == 'v4.6.1'">
       <ItemGroup>
         <Reference Include="FSharp.Core">
-          <HintPath>..\packages\FSharp.Core\lib\net40\FSharp.Core.dll</HintPath>
+          <HintPath>..\packages\FSharp.Core\lib\net45\FSharp.Core.dll</HintPath>
           <Private>True</Private>
           <Paket>True</Paket>
         </Reference>
       </ItemGroup>
     </When>
   </Choose>
-</Project>
+  <Choose>
+    <When Condition="$(TargetFrameworkIdentifier) == '.NETFramework' And $(TargetFrameworkVersion) == 'v4.6.1'">
+      <ItemGroup>
+        <Reference Include="System.ValueTuple">
+          <HintPath>..\packages\System.ValueTuple\lib\netstandard1.0\System.ValueTuple.dll</HintPath>
+          <Private>True</Private>
+          <Paket>True</Paket>
+        </Reference>
+      </ItemGroup>
+    </When>
+  </Choose>
+</Project>
\ No newline at end of file
diff --git a/Expecto.Tests.CSharp/app.config b/Expecto.Tests.CSharp/app.config
index 0af883b..b56a575 100644
--- a/Expecto.Tests.CSharp/app.config
+++ b/Expecto.Tests.CSharp/app.config
@@ -4,6 +4,6 @@
   <dependentAssembly>
     <Paket>True</Paket>
     <assemblyIdentity name="FSharp.Core" publicKeyToken="b03f5f7f11d50a3a" culture="neutral" />
-    <bindingRedirect oldVersion="0.0.0.0-65535.65535.65535.65535" newVersion="4.3.1.0" />
+    <bindingRedirect oldVersion="0.0.0.0-65535.65535.65535.65535" newVersion="4.4.1.0" />
   </dependentAssembly>
 </assemblyBinding></runtime></configuration>
diff --git a/Expecto.Tests/Expecto.Tests.fsproj b/Expecto.Tests/Expecto.Tests.fsproj
index 5426cff..9a47116 100644
--- a/Expecto.Tests/Expecto.Tests.fsproj
+++ b/Expecto.Tests/Expecto.Tests.fsproj
@@ -1,4 +1,4 @@
-<?xml version="1.0" encoding="utf-8"?>
+﻿<?xml version="1.0" encoding="utf-8"?>
 <Project ToolsVersion="4.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
   <PropertyGroup>
     <Configuration Condition=" '$(Configuration)' == '' ">Debug</Configuration>
@@ -51,6 +51,12 @@
   </Choose>
   <Import Project="$(FSharpTargetsPath)" Condition="Exists('$(FSharpTargetsPath)')" />
   <ItemGroup>
+    <Content Include="any\netstandard1.6\FSharp.Core.optdata">
+      <Paket>True</Paket>
+    </Content>
+    <Content Include="any\netstandard1.6\FSharp.Core.sigdata">
+      <Paket>True</Paket>
+    </Content>
     <Compile Include="Prelude.fs" />
     <Compile Include="Tests.fs" />
     <Compile Include="SummaryTests.fs" />
@@ -88,7 +94,7 @@
     <When Condition="$(TargetFrameworkIdentifier) == '.NETFramework' And $(TargetFrameworkVersion) == 'v4.6.1'">
       <ItemGroup>
         <Reference Include="BenchmarkDotNet">
-          <HintPath>..\packages\BenchmarkDotNet\lib\net45\BenchmarkDotNet.dll</HintPath>
+          <HintPath>..\packages\BenchmarkDotNet\lib\net46\BenchmarkDotNet.dll</HintPath>
           <Private>True</Private>
           <Paket>True</Paket>
         </Reference>
@@ -99,7 +105,7 @@
     <When Condition="$(TargetFrameworkIdentifier) == '.NETFramework' And $(TargetFrameworkVersion) == 'v4.6.1'">
       <ItemGroup>
         <Reference Include="BenchmarkDotNet.Core">
-          <HintPath>..\packages\BenchmarkDotNet.Core\lib\net45\BenchmarkDotNet.Core.dll</HintPath>
+          <HintPath>..\packages\BenchmarkDotNet.Core\lib\net46\BenchmarkDotNet.Core.dll</HintPath>
           <Private>True</Private>
           <Paket>True</Paket>
         </Reference>
@@ -110,7 +116,7 @@
     <When Condition="$(TargetFrameworkIdentifier) == '.NETFramework' And $(TargetFrameworkVersion) == 'v4.6.1'">
       <ItemGroup>
         <Reference Include="BenchmarkDotNet.Toolchains.Roslyn">
-          <HintPath>..\packages\BenchmarkDotNet.Toolchains.Roslyn\lib\net45\BenchmarkDotNet.Toolchains.Roslyn.dll</HintPath>
+          <HintPath>..\packages\BenchmarkDotNet.Toolchains.Roslyn\lib\net46\BenchmarkDotNet.Toolchains.Roslyn.dll</HintPath>
           <Private>True</Private>
           <Paket>True</Paket>
         </Reference>
@@ -132,7 +138,7 @@
     <When Condition="$(TargetFrameworkIdentifier) == '.NETFramework' And $(TargetFrameworkVersion) == 'v4.6.1'">
       <ItemGroup>
         <Reference Include="FSharp.Core">
-          <HintPath>..\packages\FSharp.Core\lib\net40\FSharp.Core.dll</HintPath>
+          <HintPath>..\packages\FSharp.Core\lib\net45\FSharp.Core.dll</HintPath>
           <Private>True</Private>
           <Paket>True</Paket>
         </Reference>
@@ -164,6 +170,28 @@
   <Choose>
     <When Condition="$(TargetFrameworkIdentifier) == '.NETFramework' And $(TargetFrameworkVersion) == 'v4.6.1'">
       <ItemGroup>
+        <Reference Include="Microsoft.DotNet.InternalAbstractions">
+          <HintPath>..\packages\Microsoft.DotNet.InternalAbstractions\lib\net451\Microsoft.DotNet.InternalAbstractions.dll</HintPath>
+          <Private>True</Private>
+          <Paket>True</Paket>
+        </Reference>
+      </ItemGroup>
+    </When>
+  </Choose>
+  <Choose>
+    <When Condition="$(TargetFrameworkIdentifier) == '.NETFramework' And $(TargetFrameworkVersion) == 'v4.6.1'">
+      <ItemGroup>
+        <Reference Include="Microsoft.DotNet.PlatformAbstractions">
+          <HintPath>..\packages\Microsoft.DotNet.PlatformAbstractions\lib\net451\Microsoft.DotNet.PlatformAbstractions.dll</HintPath>
+          <Private>True</Private>
+          <Paket>True</Paket>
+        </Reference>
+      </ItemGroup>
+    </When>
+  </Choose>
+  <Choose>
+    <When Condition="$(TargetFrameworkIdentifier) == '.NETFramework' And $(TargetFrameworkVersion) == 'v4.6.1'">
+      <ItemGroup>
         <Reference Include="System.AppContext">
           <HintPath>..\packages\System.AppContext\lib\net46\System.AppContext.dll</HintPath>
           <Private>True</Private>
@@ -413,4 +441,4 @@
       </ItemGroup>
     </When>
   </Choose>
-</Project>
+</Project>
\ No newline at end of file
diff --git a/Expecto.Tests/app.config b/Expecto.Tests/app.config
index 3a0f273..03b46bb 100644
--- a/Expecto.Tests/app.config
+++ b/Expecto.Tests/app.config
@@ -5,17 +5,17 @@
   <dependentAssembly>
     <Paket>True</Paket>
     <assemblyIdentity name="FSharp.Core" publicKeyToken="b03f5f7f11d50a3a" culture="neutral" />
-    <bindingRedirect oldVersion="0.0.0.0-65535.65535.65535.65535" newVersion="4.3.1.0" />
+    <bindingRedirect oldVersion="0.0.0.0-65535.65535.65535.65535" newVersion="4.4.1.0" />
   </dependentAssembly>
   <dependentAssembly>
     <Paket>True</Paket>
     <assemblyIdentity name="Microsoft.CodeAnalysis" publicKeyToken="31bf3856ad364e35" culture="neutral" />
-    <bindingRedirect oldVersion="0.0.0.0-65535.65535.65535.65535" newVersion="2.0.0.0" />
+    <bindingRedirect oldVersion="0.0.0.0-65535.65535.65535.65535" newVersion="2.1.0.0" />
   </dependentAssembly>
   <dependentAssembly>
     <Paket>True</Paket>
     <assemblyIdentity name="Microsoft.CodeAnalysis.CSharp" publicKeyToken="31bf3856ad364e35" culture="neutral" />
-    <bindingRedirect oldVersion="0.0.0.0-65535.65535.65535.65535" newVersion="2.0.0.0" />
+    <bindingRedirect oldVersion="0.0.0.0-65535.65535.65535.65535" newVersion="2.1.0.0" />
   </dependentAssembly>
   <dependentAssembly>
     <Paket>True</Paket>
diff --git a/Expecto.netcore/Expecto.netcore.fsproj b/Expecto.netcore/Expecto.netcore.fsproj
index 9df2b89..8531201 100644
--- a/Expecto.netcore/Expecto.netcore.fsproj
+++ b/Expecto.netcore/Expecto.netcore.fsproj
@@ -20,4 +20,5 @@
     <PackageReference Include="System.Diagnostics.FileVersionInfo" Version="4.3.0" />
   </ItemGroup>
   <!--<Import Project="..\.paket\Paket.Restore.targets" />-->
+  <Import Project="..\.paket\Paket.Restore.targets" />
 </Project>
\ No newline at end of file
diff --git a/Expecto/Expecto.fsproj b/Expecto/Expecto.fsproj
index da1be5a..e6d6f93 100644
--- a/Expecto/Expecto.fsproj
+++ b/Expecto/Expecto.fsproj
@@ -35,6 +35,12 @@
     <Reference Include="System.Numerics" />
   </ItemGroup>
   <ItemGroup>
+    <Content Include="any\netstandard1.6\FSharp.Core.optdata">
+      <Paket>True</Paket>
+    </Content>
+    <Content Include="any\netstandard1.6\FSharp.Core.sigdata">
+      <Paket>True</Paket>
+    </Content>
     <Compile Include="..\paket-files\logary\logary\src\Logary.Facade\Facade.fs">
       <Paket>True</Paket>
       <Link>paket-files/Facade.fs</Link>
@@ -75,7 +81,7 @@
     <When Condition="$(TargetFrameworkIdentifier) == '.NETFramework' And $(TargetFrameworkVersion) == 'v4.6.1'">
       <ItemGroup>
         <Reference Include="FSharp.Core">
-          <HintPath>..\packages\FSharp.Core\lib\net40\FSharp.Core.dll</HintPath>
+          <HintPath>..\packages\FSharp.Core\lib\net45\FSharp.Core.dll</HintPath>
           <Private>True</Private>
           <Paket>True</Paket>
         </Reference>
@@ -108,4 +114,15 @@
       </ItemGroup>
     </When>
   </Choose>
+  <Choose>
+    <When Condition="$(TargetFrameworkIdentifier) == '.NETFramework' And $(TargetFrameworkVersion) == 'v4.6.1'">
+      <ItemGroup>
+        <Reference Include="System.ValueTuple">
+          <HintPath>..\packages\System.ValueTuple\lib\netstandard1.0\System.ValueTuple.dll</HintPath>
+          <Private>True</Private>
+          <Paket>True</Paket>
+        </Reference>
+      </ItemGroup>
+    </When>
+  </Choose>
 </Project>
\ No newline at end of file
diff --git a/paket.lock b/paket.lock
index 62634a5..323cede 100644
--- a/paket.lock
+++ b/paket.lock
@@ -3,20 +3,24 @@ FRAMEWORK: NET461
 NUGET
   remote: https://www.nuget.org/api/v2
     Argu (3.7)
-    BenchmarkDotNet (0.10.3)
-      BenchmarkDotNet.Core (>= 0.10.3)
-      BenchmarkDotNet.Toolchains.Roslyn (>= 0.10.3)
-    BenchmarkDotNet.Core (0.10.3)
+    BenchmarkDotNet (0.10.5)
+      BenchmarkDotNet.Core (>= 0.10.5)
+      BenchmarkDotNet.Toolchains.Roslyn (>= 0.10.5)
+    BenchmarkDotNet.Core (0.10.5)
+      Microsoft.DotNet.InternalAbstractions (>= 1.0)
+      Microsoft.DotNet.PlatformAbstractions (>= 1.1.1)
       System.Threading.Tasks.Extensions (>= 4.3)
-    BenchmarkDotNet.Toolchains.Roslyn (0.10.3)
-      BenchmarkDotNet.Core (>= 0.10.3)
-      Microsoft.CodeAnalysis.CSharp (>= 1.3.2)
-      System.Threading.Tasks (>= 4.0.11)
+      System.ValueTuple (>= 4.3)
+    BenchmarkDotNet.Toolchains.Roslyn (0.10.5)
+      BenchmarkDotNet.Core (>= 0.10.5)
+      Microsoft.CodeAnalysis.CSharp (>= 2.0)
+      System.Threading.Tasks (>= 4.3)
     FsCheck (2.8)
       FSharp.Core (>= 3.1.2.5)
-    FSharp.Core (3.1.2.5)
+    FSharp.Core (4.1.12)
+      System.ValueTuple (>= 4.3)
     Microsoft.CodeAnalysis.Analyzers (1.1)
-    Microsoft.CodeAnalysis.Common (2.0)
+    Microsoft.CodeAnalysis.Common (2.1)
       Microsoft.CodeAnalysis.Analyzers (>= 1.1)
       System.AppContext (>= 4.3)
       System.Collections (>= 4.3)
@@ -56,8 +60,10 @@ NUGET
       System.Xml.XDocument (>= 4.3)
       System.Xml.XmlDocument (>= 4.3)
       System.Xml.XPath.XDocument (>= 4.3)
-    Microsoft.CodeAnalysis.CSharp (2.0)
-      Microsoft.CodeAnalysis.Common (2.0)
+    Microsoft.CodeAnalysis.CSharp (2.1)
+      Microsoft.CodeAnalysis.Common (2.1)
+    Microsoft.DotNet.InternalAbstractions (1.0)
+    Microsoft.DotNet.PlatformAbstractions (1.1.1)
     Mono.Cecil (0.10-beta5)
     NuGet.CommandLine (3.5)
     System.AppContext (4.3)
@@ -113,4 +119,4 @@ GITHUB
 GROUP build
 NUGET
   remote: https://www.nuget.org/api/v2
-    FAKE (4.57.4)
+    FAKE (4.60)
diff --git a/testdata/Expecto.Focused.Tests/Expecto.Focused.Tests.fsproj b/testdata/Expecto.Focused.Tests/Expecto.Focused.Tests.fsproj
index 5bc1d4d..e3c74d5 100644
--- a/testdata/Expecto.Focused.Tests/Expecto.Focused.Tests.fsproj
+++ b/testdata/Expecto.Focused.Tests/Expecto.Focused.Tests.fsproj
@@ -44,17 +44,13 @@
   </Choose>
   <Import Project="$(FSharpTargetsPath)" Condition="Exists('$(FSharpTargetsPath)')" />
   <Import Project="..\..\.paket\paket.targets" />
-  <Choose>
-    <When Condition="$(TargetFrameworkIdentifier) == '.NETFramework' And $(TargetFrameworkVersion) == 'v4.6.1'">
-      <ItemGroup>
-        <Reference Include="FSharp.Core">
-          <HintPath>..\..\packages\FSharp.Core\lib\net40\FSharp.Core.dll</HintPath>
-          <Paket>True</Paket>
-        </Reference>
-      </ItemGroup>
-    </When>
-  </Choose>
   <ItemGroup>
+    <Content Include="any\netstandard1.6\FSharp.Core.optdata">
+      <Paket>True</Paket>
+    </Content>
+    <Content Include="any\netstandard1.6\FSharp.Core.sigdata">
+      <Paket>True</Paket>
+    </Content>
     <Compile Include="Main.fs" />
     <None Include="paket.references" />
     <Content Include="app.config" />
@@ -69,4 +65,26 @@
       <Private>True</Private>
     </ProjectReference>
   </ItemGroup>
+  <Choose>
+    <When Condition="$(TargetFrameworkIdentifier) == '.NETFramework' And $(TargetFrameworkVersion) == 'v4.6.1'">
+      <ItemGroup>
+        <Reference Include="FSharp.Core">
+          <HintPath>..\..\packages\FSharp.Core\lib\net45\FSharp.Core.dll</HintPath>
+          <Private>True</Private>
+          <Paket>True</Paket>
+        </Reference>
+      </ItemGroup>
+    </When>
+  </Choose>
+  <Choose>
+    <When Condition="$(TargetFrameworkIdentifier) == '.NETFramework' And $(TargetFrameworkVersion) == 'v4.6.1'">
+      <ItemGroup>
+        <Reference Include="System.ValueTuple">
+          <HintPath>..\..\packages\System.ValueTuple\lib\netstandard1.0\System.ValueTuple.dll</HintPath>
+          <Private>True</Private>
+          <Paket>True</Paket>
+        </Reference>
+      </ItemGroup>
+    </When>
+  </Choose>
 </Project>
\ No newline at end of file
diff --git a/testdata/Expecto.Focused.Tests/app.config b/testdata/Expecto.Focused.Tests/app.config
index 0af883b..b56a575 100644
--- a/testdata/Expecto.Focused.Tests/app.config
+++ b/testdata/Expecto.Focused.Tests/app.config
@@ -4,6 +4,6 @@
   <dependentAssembly>
     <Paket>True</Paket>
     <assemblyIdentity name="FSharp.Core" publicKeyToken="b03f5f7f11d50a3a" culture="neutral" />
-    <bindingRedirect oldVersion="0.0.0.0-65535.65535.65535.65535" newVersion="4.3.1.0" />
+    <bindingRedirect oldVersion="0.0.0.0-65535.65535.65535.65535" newVersion="4.4.1.0" />
   </dependentAssembly>
 </assemblyBinding></runtime></configuration>
```

Pinging @forki

you may not have noticed this, but you have paket hardcoded in the build.cmd and build.sh to `4.0.0-alpha030`

when I run it on my machine it uses the wrong msbuild 
```
Starting Target: Build (==> Clean)
Building project: C:\Users\jared\Github\Forks\expecto\Expecto.sln
  C:\Program Files (x86)\Microsoft Visual Studio\2017\Community\MSBuild\15.0\Bin\MSBuild.exe  C:\Users\jared\Github\Forks\expecto\Expecto.sln /t:Rebuild /m      /p:RestorePackages="False" /p:Configuration="Release" /logger:Fake.MsBuildLogger+ErrorLogger,"C:\Users\jared\Github\Forks\expecto\packages\build\FAKE\tools\FakeLib.dll"
Microsoft (R) Build Engine version 15.1.1012.6693
Copyright (C) Microsoft Corporation. All rights reserved.
```
I add those lines from before to `build.fsx`

```fsharp
let msbuild14 = ProgramFilesX86</>"MSBuild"</>"14.0"</>"Bin"</>"MSBuild.exe"

if isWindows && fileExists msbuild14 then
    setEnvironVar "MSBUILD"  msbuild14
```
```
       CopyFilesToOutputDirectory:
         Copying file from "obj\Release\Expecto.dll" to "bin\Release\Expecto.dll".
         Expecto -> C:\Users\jared\Github\Forks\expecto\Expecto\bin\Release\Expecto.dll
         Copying file from "obj\Release\Expecto.pdb" to "bin\Release\Expecto.pdb".
     2>Done Building Project "C:\Users\jared\Github\Forks\expecto\Expecto\Expecto.fsproj" (Rebuild target(s)).
```
no manifest resource problem

the build still fails though with
```
       "C:\Users\jared\Github\Forks\expecto\Expecto.sln" (Rebuild target) (1) ->
       "C:\Users\jared\Github\Forks\expecto\Expecto.Tests.CSharp\Expecto.Tests.CSharp.csproj" (Rebuild target) (8) ->
       (CoreCompile target) ->
         Tests.cs(3,7): error CS0246: The type or namespace name 'Expecto' could not be found (are you missing a using d
       irective or an assembly reference?) [C:\Users\jared\Github\Forks\expecto\Expecto.Tests.CSharp\Expecto.Tests.CShar
       p.csproj]
         Tests.cs(4,7): error CS0246: The type or namespace name 'Expecto' could not be found (are you missing a using d
       irective or an assembly reference?) [C:\Users\jared\Github\Forks\expecto\Expecto.Tests.CSharp\Expecto.Tests.CShar
       p.csproj]
         Tests.cs(10,10): error CS0246: The type or namespace name 'TestsAttribute' could not be found (are you missing
       a using directive or an assembly reference?) [C:\Users\jared\Github\Forks\expecto\Expecto.Tests.CSharp\Expecto.Te
       sts.CSharp.csproj]
         Tests.cs(10,10): error CS0246: The type or namespace name 'Tests' could not be found (are you missing a using d
       irective or an assembly reference?) [C:\Users\jared\Github\Forks\expecto\Expecto.Tests.CSharp\Expecto.Tests.CShar
       p.csproj]
         Tests.cs(11,23): error CS0246: The type or namespace name 'Expecto' could not be found (are you missing a using
        directive or an assembly reference?) [C:\Users\jared\Github\Forks\expecto\Expecto.Tests.CSharp\Expecto.Tests.CSh
       arp.csproj]
```
but that's probably due to some other configuration issue

like I said before, paket is not the source of this issue

Not solved;

```

         	Specified references-file '/Users/h/dev/haf/expecto/Expecto.netcore/obj/Expecto.netcore.fsproj.references' does not exist.
     1>/Users/h/dev/haf/expecto/.paket/Paket.Restore.targets(59,5): error MSB3073: The command "/Library/Frameworks/Mono.framework/Commands/mono --runtime=v4.0.30319 "/Users/h/dev/haf/expecto/.paket/paket.exe" fix-nuspec file "obj/Expecto.5.0.0-beta3.nuspec" references-file "/Users/h/dev/haf/expecto/Expecto.netcore/obj/Expecto.netcore.fsproj.references" " exited with code 1. [/Users/h/dev/haf/expecto/Expecto.netcore/Expecto.netcore.fsproj]
     1>Done Building Project "/Users/h/dev/haf/expecto/Expecto.netcore/Expecto.netcore.fsproj" (pack target(s)) -- FAILED.
```

when rake runs `dotnet pack -v n --no-build -c Release -o ../build/clipkg /p:Version=5.0.0-beta3 Expecto.netcore/Expecto.netcore.fsproj` from running `bundle exec rake` to start with.

Back to 
/usr/local/share/dotnet/sdk/1.0.1/Microsoft.Common.CurrentVersion.targets(2628,7): error MSB4057: The target "CreateManifestResourceNames" does not exist in the project. [/Users/h/dev/haf/expecto/Expecto.netcore/Expecto.netcore.fsproj]
    0 Warning(s)
    1 Error(s)

> Need to stop fighting windmills for today. /cc @enricosada

Atm (with .net core sdk 1.0), you need to do at least once the `dotnet restore`, before `dotnet build`
And you are also responsible to do that manually if you change `PackageReference` 
see referenced issue for more info

> Back to
> /usr/local/share/dotnet/sdk/1.0.1/Microsoft.Common.CurrentVersion.targets(2628,7): error MSB4057: The target "CreateManifestResourceNames" does not exist in the project.

That happen if prohect is not restored. i know. i was too short on time for 1.0 and didnt see that. i cannot fix it untile next 1.0 or 2.0 release (it's a file embedded in dotnet cli, cannot update it)

added https://github.com/dotnet/netcorecli-fsc/issues/109 to track that

btw @haf @cloudRoutine feel free to open bug in https://github.com/dotnet/netcorecli-fsc/ (fsharp) or https://github.com/dotnet/cli/ if you see it

--- a/Expecto.netcore.Tests/Expecto.netcore.Tests.fsproj
+++ b/Expecto.netcore.Tests/Expecto.netcore.Tests.fsproj
@@ -31,5 +31,4 @@
     <ProjectReference Include="../Expecto.netcore/Expecto.netcore.fsproj" />
     <PackageReference Include="FsCheck" Version="3.0.0-alpha-b752" />
   </ItemGroup>
-  <!--<Import Project="..\.paket\Paket.Restore.targets" />-->
 </Project>

--- a/Expecto.netcore/Expecto.netcore.fsproj
+++ b/Expecto.netcore/Expecto.netcore.fsproj
@@ -19,6 +19,4 @@
     <PackageReference Include="System.Diagnostics.TraceSource" Version="4.3.0" />
     <PackageReference Include="System.Diagnostics.FileVersionInfo" Version="4.3.0" />
   </ItemGroup>
-  <!--<Import Project="..\.paket\Paket.Restore.targets" />-->
-  <Import Project="..\.paket\Paket.Restore.targets" />
 </Project>

--- a/Rakefile
+++ b/Rakefile
@@ -62,7 +62,7 @@ build :compile => [:versioning, :restore, :assembly_info, :build_dotnetcli] do |
   b.sln = 'Expecto.sln'
 end
 
-task :build_dotnetcli => [:versioning, :restore_dotnetcli, :assembly_info] do
+task :build_dotnetcli => [:versioning, :restore, :assembly_info] do
   system "dotnet", %W|build -c #{Configuration} -f netstandard1.6 Expecto.netcore/Expecto.netcore.fsproj|
 end
 

--- a/appveyor.yml
+++ b/appveyor.yml
@@ -9,7 +9,7 @@ install:
   - bundle install
 
 build_script:
-  - cmd: build.cmd "All"
+  - cmd: build.cmd
 
 #  # .NET Core SDK binaries
 #  - ps: $url = "https://dotnetcli.blob.core.windows.net/dotnet/Sdk/rel-1.0.0/dotnet-dev-win-x64.latest.zip"

--- a/appveyor.yml
+++ b/appveyor.yml
@@ -7,6 +7,7 @@ environment:
 install:
   - set PATH=C:\Ruby23-x64\bin;%PATH%
   - bundle install
+  - dotnet --version
 
 build_script:
   - cmd: build.cmd
@@ -24,8 +25,8 @@ build_script:
 #  - dotnet --info
 #  - cmd: build.cmd
 
-# artifacts:
-#  - path: 'bin'
+artifacts:
+ - path: 'build\pkg'
 
 test: off
 version: 0.0.1.{build}

--- a/build.cmd
+++ b/build.cmd
@@ -1,4 +1,4 @@
 @echo off
 
-bundle
-bundle exec rake %*
+call bundle
+call bundle exec rake %*

